packages/orbit-components/src/Loading/__examples__/LOADING_CHILDREN.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import * as React from "react";
+
+import Button from "../../Button";
+import Loading from "../index";
+import Stack from "../../Stack";
+import Text from "../../Text";
+
+export default {
+  Example: () => {
+    const [loadingNow, setLoadingNow] = React.useState(true);
+    return (
+      <Stack direction="column">
+        <Loading loading={loadingNow} type="inlineLoader" text="Saving your request">
+          <Text>Your request has been saved!</Text>
+        </Loading>
+        <Loading loading={loadingNow} type="inlineLoader" text="Saving your request" />
```

Comment:
Isn't this redundant, or did you want to really have it there @CollierCZ? 🙂

Reply:
The point is that the Loading component without children is always shown, even when you toggle the state. So it was intentional.

Reply:
Okay, fair point! 🙂